package.json

```diff
@@ -14,6 +14,7 @@
     "update-api": "yarn add github:bcgov/jag-shuber-api"
   },
   "dependencies": {
+    "add": "^2.0.6",
```

Comment:
I think this dependency was added in error.  This combined with the new dependency on `yarn` makes it seem like perhaps some typing occured while a process was executing in the terminal i.e.

`yarn add yarn add` would add dependencies on `yarn` and `add`